internal/git/releases.go

```diff
@@ -18,11 +18,14 @@ import (
 //go:embed goreleaser.yml
 var tfGoReleaserConfig string
 
-func (g *Git) SetReleaseToPublished(version string) error {
+func (g *Git) SetReleaseToPublished(version, directory string) error {
 	if g.repo == nil {
 		return fmt.Errorf("repo not cloned")
 	}
 	tag := "v" + version
+	if directory != "" && directory != "." && directory != "./" {
```

Comment:
Can you walk me through how this is intended to be used in this context? we do want to use directories as tags?

Reply:
This is how it already works today. Not changing anything

Reply:
I'm just making sure reading a release that was already created doesn't fail

Reply:
see https://github.com/codatio/client-sdk-typescript/releases